package.json

```diff
@@ -161,28 +153,38 @@
     "not Safari < 10",
     "not ios_saf < 10"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "@babel/polyfill": "^7.10.4",
+    "core-js": "^3.6.5",
+    "regenerator-runtime": "^0.13.7"
+  },
   "devDependencies": {
+    "@babel/cli": "^7.10.5",
```

Comment:
Are you sure this one is necessary? I wouldn't think so if doing through `@rollup/plugin-babel`.

Reply:
No. I don't think we need it. I was using it for tests. Please don't forget that this branch is far from being good to merge.

Reply:
removed